corehq/apps/data_interfaces/templates/data_interfaces/auto_update_rules.html

```diff
@@ -27,12 +28,16 @@
     <div class="spacer"></div>
     <ul class="nav nav-tabs">
       <li class="active"><a data-toggle="tab" href="#tabs-update-rules">{% trans "Automatic Update Rules" %}</a></li>
+      <li><a data-toggle="tab" href="#tabs-rule-run-history">{% trans "Update Rule Run History" %}</a></li>
```

Comment:
```suggestion
      <li><a data-toggle="tab" href="#tabs-rule-run-history">{% trans "Rule Run History" %}</a></li>
```

Reply:
^ optional nitpick - it's confusing that we call these "update rules" because "update rule" sounds like an action you can take

Reply:
I appreciate this nitpick a lot. I suppose the confusion starts when you take away the preceding "Automatic" or "Case" and just leave "Update Rule". I also haven't loved the name "Rule Run" but maybe I'm just not a fan of alliteration. I'll make that change 👍🏻

Reply:
See 0f142f0e12c129efd510a414f5b0990efcfee653

Reply:
thank you for updating the template file name, too :)